example/eject.js

```diff
@@ -16,20 +17,20 @@
 
 'use strict';
 
-const mountutils = require('.');
-const drive = process.argv[2];
+const mountUtils = require('..');
 
-if (!drive) {
-  console.error(`Usage: ${process.argv[0]} ${process.argv[1]} <drive>`);
+let argv = process.argv.slice(2);
+let disk = argv.shift();
+
+if (!disk) {
+  console.error(`Usage: node example/eject <drive>`);
   process.exit(1);
 }
 
-console.log(`Unmounting ${drive}`);
+process.env.MOUNTUTILS_DEBUG = true;
 
-mountutils.unmountDisk(drive, (error) => {
-  if (error) {
-    throw error;
-  }
+console.log('Ejecting', disk, '...')
```

Comment:
And another minor question: Any reason for using `console.log('Ejecting', disk, '...')` when it seems like everywhere else we do this as ``console.log(`Ejecting ${disk} ...`)`` ?